<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="comboBox"/>
    </item>
    <item>
     <widget class="QPushButton" name="Connect_Button">
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Ping_Button">
      <property name="text">
       <string>PING</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Jump_Button">
      <property name="text">
       <string>JUMP</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Verify_Button">
      <property name="text">
       <string>VERIFY</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Write_Button">
      <property name="text">
       <string>WRITE</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Erase_Button">
      <property name="text">
       <string>ERASE</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Read_Button">
      <property name="text">
       <string>READ</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>733</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
